tests/acceptance/04_examples/check_syntax.cf

```diff
@@ -0,0 +1,95 @@
+#######################################################
+#
+# Check the example syntax
+#
+#######################################################
+
+body common control
+{
+      inputs => { "../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+}
+
+#######################################################
+
+bundle agent init
+{
+  vars:
+      "basedir" string => "$(G.cwd)/../../examples";
+      "all_examples" slist => findfiles("$(basedir)/*.cf");
+
+  reports:
+    DEBUG_LOTS::
+      "$(this.bundle): will consider example $(all_examples)";
+}
+
+#######################################################
+
+bundle agent test
+{
+  meta:
+      # It's unrealistic to keep this test perfectly stable on all non-Linux platforms.
+      # It would require examples that quickly get very cluttered.
+      "test_skip_unsupported" string => "!linux";
```

Comment:
Please add a `"description" string => "Syntax check each policy (.cf) file in examples";` meta var.

Reply:
Added.